BootloaderCorePkg/Tools/BuildUtility.py

```diff
@@ -571,7 +571,11 @@ def gen_payload_bin (fv_dir, pld_list, pld_bin, priv_key, brd_name = None):
 		else :
 			dst_path = os.path.join(fv_dir, os.path.basename(src_file))
 
-		shutil.copy (src_file, dst_path)
+		if not os.path.exists(src_file):
+			raise Exception ("Can not fine payload file '%s' !" % src_file)
```

Comment:
I think it should be "Cannot find" instead of "Can not fine"

Reply:
Good catch.  :)  Will change it.